lnbits/wallets/alby.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+from typing import AsyncGenerator, Optional
+
+import httpx
+from loguru import logger
+
+from lnbits.settings import settings
+
+from .base import (
+    InvoiceResponse,
+    PaymentResponse,
+    PaymentStatus,
+    StatusResponse,
+    Unsupported,
+    Wallet,
+)
+
+
+class AlbyWallet(Wallet):
+    """https://guides.getalby.com/alby-wallet-api/reference/api-reference"""
+
+    def __init__(self):
+        endpoint = settings.alby_api_endpoint
+
+        if not endpoint or not settings.alby_access_token:
+            raise Exception("cannot initialize getalby")
+
+        self.endpoint = endpoint[:-1] if endpoint.endswith("/") else endpoint
+        self.auth = {
+            "Authorization": "Bearer " + settings.alby_access_token,
+            "User-Agent": "LNbits/0.12",
```

Comment:
We can use the version from settings:

```suggestion
            "User-Agent": f"LNbits/{settings.version}",
```